source/install/install-ubuntu-1604-server.rst

```diff
@@ -4,8 +4,7 @@ Installing Ubuntu Server 16.04 LTS
 ==================================
 
 .. warning::
-  Ubuntu 16.04 is approaching its End of Life in April 2021.
-  Because of this, we don't recommend installing new instances of Mattermost on Ubuntu 16.04. Rather use a newer version such as Ubuntu 20.04.
+   Ubuntu 16.04 is approaching its End of Life in April 2021. Because of this, we don't recommend installing new instances of Mattermost on Ubuntu 16.04. Please   refer to install guides for :doc:`Ubuntu 18.04 LTS <install-ubuntu-1804>` or :doc:`Ubuntu 20.04 LTS <install-ubuntu-2004>` instead.
```

Comment:
```suggestion
   Ubuntu 16.04 is approaching its End of Life in April 2021. Because of this, we don't recommend installing new instances of Mattermost on Ubuntu 16.04. Please refer to install guides for :doc:`Ubuntu 18.04 LTS <install-ubuntu-1804>` or :doc:`Ubuntu 20.04 LTS <install-ubuntu-2004>` instead.
```